toonz/sources/include/toonzqt/lineedit.h

```diff
@@ -33,6 +33,7 @@ class DVAPI LineEdit : public QLineEdit {
 
   bool m_isReturnPressed;
   bool m_forbiddenSpecialChars;
+  bool m_mouseDragEditing;
```

Comment:
@turtletooth Sorry for the late notice, but it seems that `bool m_mouseDragEditing` should be initialized to `false` here or in ctor.